test/unit/mocked_data/test_get_ntp_peers/normal/expected_result.json

```diff
@@ -0,0 +1,8 @@
+{
+	"2001:DB8:0:0:8:800:200C:417A": {},
+	"17.72.148.53": {},
+	"192.168.0.1": {},
+	"37.187.56.220": [{
```

Comment:
@t0mmetje to pass the tests, please remove this and let this peer have an empty dictionary as the other ones.